ui/src/components/grid-generator/QasGridGenerator.vue

```diff
@@ -78,7 +78,7 @@ export default {
       const result = extend(true, {}, this.result)
 
       for (const key in result) {
-        if (this.formattedFields[key]?.type) {
+        if (this.formattedFields[key]?.type && !!result[key]) {
```

Comment:
Por que precisou converter para boolean? apenas `result[key]` resolve.

Reply:
result[key] pode ser null, explicitar um boolean ali acho que evita o cenario `if(true && null)` que eh null e nao entra no If tambem. 
Verificar alguma coisa + null faz sentido?

Reply:
O tableGenerator esta um pouco diferente. Como eu estou tratando as tabelas no CRM nao precisei corrigir esse problema agora, mas ja vou abrir PR disso tambem

Reply:
Mas null ou false da no mesmo na tratativa do if, se for null converte para false, se for string converte para true, estamos fazendo uma conversão desnecessaria neste caso. Sobre o bug no tablegenerator acho importante já tratar neste PR para conseguir mantém o projeto mais atualizado possível.

Reply:
Bom, sendo assim deixo `result[key]` mesmo. Ja subo as alteracoes correspondentes ao table